blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -65,6 +65,20 @@ public void init() throws IOException {
         return Collections.unmodifiableMap(props);
     }
 
+    public Map<Object, Object> getAllProperties() {
```

Comment:
If thats the case, may be we should fix getProperties, instead of creating a new method? I am fine either way I guess but can you atleast add some documentation to highlight the difference between the two?